opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -366,11 +352,13 @@ def __init__(
         instrumentation_info: "InstrumentationInfo",
     ):
         self.instrumentation_info = instrumentation_info
-        self.batcher = UngroupedBatcher(source.stateful)
+        self.batcher = Batcher(source.stateful)
         self.resource = source.resource
         self.metrics = set()
         self.observers = set()
+        self.metrics_lock = threading.Lock()
         self.observers_lock = threading.Lock()
+        self.view_manager = ViewManager()
```

Comment:
Should every `Record` have a separate `ViewManager`? I thought the only `ViewManager` would be the one attached to the `Meter`, what am I missing here?

Reply:
this is on the `Meter`, Github just hid that part so it looks like its on Record 😛